packages/neutrino-lint-base/src/index.js

```diff
@@ -37,4 +40,27 @@ module.exports = ({ config }) => {
 
   config.resolve.modules.add(MODULES);
   config.resolveLoader.modules.add(MODULES);
+
+  neutrino.custom.eslintrc = () => {
+    const options = clone(config.module.rule('lint').loaders.get('eslint').options);
+
+    options.extends = options.baseConfig.extends;
+    options.useEslintrc = true;
+    options.env = options.envs.reduce((env, key) => {
+      env[key] = true;
+      return env;
```

Comment:
@eliperelman Out of curiosity, in what scenario do you not prefer adding an empty line between an assignment and the return value.